app/views/admin/content_page_versions/_form.html.erb

```diff
@@ -26,7 +26,8 @@
           <% if action_name === "edit" || action_name === "new" || action_name === "update" %>
             <div class="fixed-footer">
               <%= form.govuk_submit "Save as draft" %>
-              <%= link_to 'Cancel', versions_content_page_path(@content_page_version.content_page), data: { confirm: "Any changes will be lost, are you sure ?" }, class: "govuk-link" %>
+              <%= link_to 'Cancel', edit_admin_content_page_content_page_version_path(@content_page_version.content_page, @content_page_version), :data => {:confirm => "Are you sure you want to undo your changes ?", :class => "govuk-link"}%> |
+              <%= link_to 'Back', versions_admin_content_page_path(@content_page_version.content_page), :data => {:confirm => "Any changes will be lost, are you sure ?", :class => "govuk-link"}%>
```

Comment:
There was a ticket that removed the second button: https://github.com/DFE-Digital/early-years-foundation-reform/pull/530/files

This change reintroduces it. This needs to be fixed before this is merged.